coding-for-reusability/Exercise1/solution/Solution.tsx

```diff
@@ -0,0 +1,19 @@
+// type Pokemon = {
```

Comment:
Thanks, I was getting some typescript errors because the functions/types were already defined in the problems. I couldn't figure out how to not export them at first.